cmd/kubeadm/app/cmd/phases/certs_test.go

```diff
@@ -0,0 +1,267 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/renstrom/dedent"
+	"github.com/spf13/cobra"
+
+	_ "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/install"
+
```

Comment:
@lukas, this line is for triggering api machinery startup when running unit tests (which is necessary e.g. for triggering the defaulting of parameters).
I added a comment in the code for making this explicit.